src/openloco/game_commands.h

```diff
@@ -190,7 +190,7 @@ namespace openloco::game_commands
     {
         registers regs;
         regs.bl = bl; // [ 1 = remove industry]
-        regs.edi = industryId;
+        regs.dx = industryId;
```

Comment:
I'm not sure on this the function definitely checks di. dx is clobered early on in the function as well.